install/helm/agones/values.yaml

```diff
@@ -35,7 +35,7 @@ agones:
   serviceaccount:
     allocator:
       name: agones-allocator
-      annotations: {}
+      labels: {}
```

Comment:
We need both annotations and labels please.

Also, we'll need updates to the documentation as well.

Reply:
I included both..